src/main/java/org/example/internetshop/controller/cart/ShoppingCartController.java

```diff
@@ -6,20 +6,22 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import org.example.internetshop.lib.Injector;
 import org.example.internetshop.model.Product;
 import org.example.internetshop.service.ShoppingCartService;
 
 public class ShoppingCartController extends HttpServlet {
```

Comment:
```suggestion
public class Get/ShowShoppingCartController extends HttpServlet {
```
я б додав до назви get/show щоб було по назві зрозуміліше, що ми хочемо в цьому контроллері зробити з корзиною